src/answers-umd.js

```diff
@@ -731,6 +730,7 @@ class Answers {
       this.core.init({ visitor: visitor });
     } else {
       console.error(`Invalid visitor. Visitor was not set because "${JSON.stringify(visitor)}" does not have an id.`);
+      this.core.init();
```

Comment:
Needed this change to fix a test. I asked Nidhi (she originally implemented this visitor feature) and she expected it to init here.